src/tests/test_pyodide.py

```diff
@@ -502,8 +506,8 @@ def h():
               File "<exec>", line 6 in g
               File "<exec>", line 4 in f
               File "<exec>", line 9 in <module>
-              File "/lib/python3.8/site-packages/pyodide/_base.py", line 242 in run
-              File "/lib/python3.8/site-packages/pyodide/_base.py", line 344 in eval_code
+              File "/lib/pythonxxx/site-packages/pyodide/_base.py", line 242 in run
+              File "/lib/pythonxxx/site-packages/pyodide/_base.py", line 344 in eval_code
```

Comment:
I don't understand this change. Shouldn't the path be `/lib/python3.9`?

Reply:
It's better not to have to change the number in too many places every time we update, so I changed `strip_stack_trace` to remove the Python version number from the stack trace.

Reply:
I agree changing all those paths is a bit annoying but `strip_stack_trace` is also fairly surprising. People expect to be able to open the files that are shown in tracebacks, while with this option those files don't exist. So in terms of usability it's not ideal.

Reply:
It only happens in this one test though?

Reply:
Plus it refers to a file in the emscripten virtual file system, so it can't necessarily be found in this location on the host operating system...

Reply:
Aww, for this test sure. I haven't looked at the diff. Nevermind.